src/components/proofs/EmailProof.tsx

```diff
@@ -67,7 +68,9 @@ export default function () {
     }
   }
 
-  async function onGenerateProof(secret?: string) {
+  async function onGenerateProof(secret: string) {
+    if (!checkDomainToken(secret)) return // TODO: Show error state here
```

Comment:
We can take out the comments for PRs

Reply:
Sure, but in the next PR that also is reviewed, it will go